habitat/config/ret.prod.conf

```diff
@@ -29,9 +29,9 @@ ret.Elixir.Ret.MediaResolver.imgur_client_id = "{{ cfg.resolver.imgur_client_id
 ret.Elixir.Ret.MediaResolver.google_poly_api_key = "{{ cfg.resolver.google_poly_api_key }}"
 ret.Elixir.Ret.MediaResolver.sketchfab_api_key = "{{ cfg.resolver.sketchfab_api_key }}"
 ret.Elixir.Ret.PageOriginWarmer.page_origin = "{{ cfg.pages.page_origin }}"
-ret.Elixir.Ret.Uploads.storage_path = "{{ cfg.uploads.storage_path }}"
-ret.Elixir.Ret.Uploads.ttl = {{ cfg.uploads.ttl }}
-ret.Elixir.Ret.Uploads.host = "{{ cfg.uploads.host }}"
+ret.Elixir.Ret.Storage.storage_path = "{{ cfg.uploads.storage_path }}"
```

Comment:
note `uploads` here is still burned in to the habitat toml configs, we'll probably want to rename these at some point but didn't want the ops creep atm